cns/mdnc_azure-cns.yaml

```diff
@@ -0,0 +1,207 @@
+apiVersion: v1
```

Comment:
I think this should probably be in `.pipelines/mdnc` or something like that instead of in the CNS directory